srcpkgs/intel-media-driver/template

```diff
@@ -1,6 +1,6 @@
 # Template file for 'intel-media-driver'
 pkgname=intel-media-driver
-version=20.4.1
+version=20.4.5
```

Comment:
There is a https://github.com/intel/media-driver/releases/tag/intel-media-21.1.0 release out, but I'm not sure it's stable, so going to skip it for now.